packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.styles.tsx

```diff
@@ -0,0 +1,142 @@
+import {
+  FontSizes,
+  HighContrastSelector,
+  ScreenWidthMaxMedium,
+  ScreenWidthMaxSmall,
+  AnimationVariables,
+  normalize
+} from '../../Styling';
+import { SearchBoxBase } from './SearchBox.base';
+import { ISearchBoxProps, ISearchBoxStyleProps, ISearchBoxStyles } from './SearchBox.types';
+
+export function getStyles(props: ISearchBoxStyleProps): ISearchBoxStyles {
+  const { theme, underlined, disabled, hasFocus } = props;
+  const { palette, fonts } = theme;
+
+  return {
+    root: [
+      'ms-Searchbox',
+      fonts.medium,
+      normalize,
+      {
+        color: palette.neutralPrimary,
+        display: 'flex',
+        flexDirection: 'row',
+        flexWrap: 'nowrap',
+        alignItems: 'stretch',
+        padding: '0 0 0 8px',
```

Comment:
Does the padding automatically correct for rtl? If so that's really cool.

Reply:
It does!